unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/IParcelScene.cs

```diff
@@ -37,5 +37,7 @@ public interface IParcelScene
         void GetWaitingComponentsDebugInfo();
         void SetEntityParent(long entityId, long parentId);
         void RemoveEntity(long id, bool removeImmediatelyFromEntitiesList = true);
+        void SetInitMessagesDone();
```

Comment:
Let's call it `MarkInitMessagesDone` or something else, for me `Set` implies that you can enable/disable a flag or similar.

Reply:
This is not super clean, but as discussed I cannot think of any other way without  refactoring the whole system and ParcelScene and that would be dangerous AF.